poly-synth/src/poly-synth.js

```diff
@@ -60,6 +62,22 @@ class PolySynth {
     this.minQ = -50;
     this.maxQ = 50;
 
+    // The bitcrusher defaults non-information reducing parameters.
```

Comment:
They are set that way, but maximum and minimum values still need to be set explicitly for the sliders. I took out the comment since it is unnecessary.